msi/release.sh

```diff
@@ -152,8 +152,10 @@ die "Could not generate file list"
 # Write the GitComponents.wxs file containing the file list
 SCRIPT_WINPATH="$(cd "$SCRIPT_PATH" && pwd -W | tr / \\\\)"
 BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
+BIND_IMAGE_PATHS='[INSTALLFOLDER]usr\\bin;[MingwFolder]bin;[System$(var.SixtyFourBit)Folder]'
```

Comment:
Hmm. Can't we define some constant, and then use that constant in the `<File>` tags instead of repeating this awfully long text everywhere?